app/controllers/reporting/reports_controller.rb

```diff
@@ -0,0 +1,27 @@
+module Reporting
+  class ReportsController < ReportingController
```

Comment:
What is `Reporting` in this context?

So are we going to have things other than Reports inherting from the Reporting Controller?

If so I can understand the idea of having "reporting" as a more abstract idea in the application. 

But I think this is a little confusing as well.

If we're not expecting anything other than reports to inherit from this - perhaps we could go with `BaseReportsController` 

Still makes it hard to understand why `ReportsController` would inherit from it though as that too seem like a generic name... 🤔